platform/ios/src/MGLMapView.mm

```diff
@@ -250,6 +250,8 @@ @interface MGLMapView () <UIGestureRecognizerDelegate,
 @property (nonatomic) CFTimeInterval frameTime;
 @property (nonatomic) CFTimeInterval averageFrameTime;
 
+@property (nonatomic, getter=isCameraAltitudeAffectedByPitch) BOOL cameraAltitudeAffectedByPitch;
```

Comment:
#12966 proposes an alternative to this property that avoids implicitly reinterpreting existing properties.

Reply:
@d-prukop in reference to the above https://github.com/mapbox/mapbox-gl-native/pull/12966#issuecomment-425213127, assuming that lands first will this PR require a change?